example/flutter_example/README.md

```diff
@@ -0,0 +1,16 @@
+# flutter_example
+
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```

Comment:
Could you replace the automatically generated README by a brief description of this example and some instructions for how to run it (similar to how I did for the other examples)? For example, assuming I've got the Flutter SDK completely setup, what are the commands that I'd have to type on the command line in order to start the example on my phone or an emulator?